components/organisms/Sidebar/Sidebar.module.css

```diff
@@ -0,0 +1,74 @@
+.sidebar-container {
+  display: flex;
+  position: fixed;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 100%;
+  bottom: 0rem;
+  height: 3.5rem;
+  background-color: var(--white);
+  border-radius: 2rem 2rem 0 0;
+  box-shadow: 0.25rem 0.063rem 0.25rem 0.188rem #7b7b7b33;
```

Comment:
Add this shadow to a variant.

Reply:
There is already a ticket to solve and put all this in variables